### DISCUSSION TITLE
how to change left menu order ?

### DISCUSSION THREAD

**Discussion by magic-thomas** [filamentphp/filament#6993] [Help] [answered]
Hello. 

How can I change the left menu list order ? 

I have read 'https://filamentphp.com/docs/2.x/admin/navigation' but I could not find how. 

What is 'navigationSort' for ? It does not affect to the list order. 

Following 'https://filamentphp.com/docs/2.x/admin/navigation#ordering-navigation-groups', the group order could be changed. 

But where can I change the group order? 

Even though I have enough experiences with Laravel and Livewire, it is so hard to catch the sense of Filament with documentation only.

**Comment by pboivin** [comment, marked as answer]
You can add `$navigationSort` on each resource class
https://filamentphp.com/docs/2.x/admin/resources/getting-started#sorting-navigation-items
